Quiz.json

```diff
@@ -0,0 +1,58 @@
+
```

Comment:
the team never used json files, we barely know them, so either avoid them or give a simple illustration to us why and when should we use them

Reply:
alright